lib/cursor.js

```diff
@@ -749,6 +596,41 @@ var nextFunction = function(self, callback) {
   }
 };
 
+function buildFindCommandQueryOptions(cursor) {
+  const queryOptions = {};
+
+  // If we have a raw query decorate the function
+  if (cursor.options.raw || cursor.cmd.raw) {
+    queryOptions.raw = cursor.options.raw || cursor.cmd.raw;
```

Comment:
I know this was here before, but what happens if `cursor.options.raw` and `cursor.cmd.raw` are set to conflicting values? Should we always be setting `queryOptions.raw` to `true` if at least one of those is `true`?